apps/api/src/app/integrations/e2e/update-integration.e2e.ts

```diff
@@ -79,4 +79,39 @@ describe('Update Integration - /integrations/:integrationId (PUT)', function ()
     expect(secondProviderIntegration.active).to.equal(true);
     expect(irrelevantProviderIntegration.active).to.equal(true);
   });
+
+  it('should update custom SMTP integration with TLS options successfully', async function () {
+    const nodeMailerProviderPayload = {
+      providerId: 'nodemailer',
+      channel: 'email',
+      credentials: {
+        host: 'smtp.example.com',
+        port: '587',
+        secure: 'true',
+        requireTls: true,
+        tlsOptions: { rejectUnauthorized: false },
+      },
+      active: true,
+      check: false,
+    };
+
+    // create integration
+    const nodeMailerIntegrationId = (await session.testAgent.post('/v1/integrations').send(nodeMailerProviderPayload))
+      .body.data._id;
+
+    // update integration
+    await session.testAgent.put(`/v1/integrations/${nodeMailerIntegrationId}`).send(nodeMailerProviderPayload);
```

Comment:
For the update, change the payload update so we can check the changes have been made. For example, change `requireTls` to false and change `tlsOptions` object. That should be enough to test the update. 👍🏻